bootstrap.sh

```diff
@@ -38,7 +38,7 @@ if [ "$1" == "--importLocales" ]; then
 fi
 
 # Download the nimbus-fml.sh script from application-services.
-NIMBUS_FML_FILE=./nimbus.fml.yaml
+NIMBUS_FML_FILE=./firefox-ios/nimbus.fml.yaml
```

Comment:
The nimbus bootstrap script downloads the nimbus-fml.sh to the `$(pwd)/bin`. You can either wrap the `curl | bash`  with a `pushd firefox-ios`/`popd` combination (that changes the directory), or you can pass a `--directory` option to the script:

```sh
curl --proto '=https' --tlsv1.2 -sSf https://raw.githubusercontent.com/mozilla/application-services/main/components/nimbus/ios/scripts/bootstrap.sh \
    | bash -s -- --directory ./firefox-ios/bin $NIMBUS_FML_FILE
```

Reply:
The firefox-ios `project.projpbx` calls the `$(SOURCE_DIR)/bin/nimbus-fml.sh`, so we need to have `nimbus-fml.sh` in the `bin` directory relative to the `firefox-ios` directory.

The nimbus bootstrap downloads the `nimbus-fml.sh`, so as long as we're able to run the fxios `bootstrap.sh`, you can remove the repository version of `nimbus-fml.sh`.

The nimbus bootstrap also tries to make a `.gitignore` in the current directory, which you will probably want to check in.

In summary, I think from a nimbus point of view: 

1. You should change the fxios `bootstrap.sh` script as above, 
2. move the contents of `./bin/nimbus*.sh` into `./firefox-ios/bin` 
3. remove `nimbus-fml.sh` (and check the deletion in to the git)
4. check in the newly formed `.gitignore` file in to git.

As for the rest of it: this is a mammoth under-taking. Good job diving in and doing it! Thank you so much!